spec/system/admin/order_spec.rb

```diff
@@ -985,26 +990,54 @@ def new_order_with_distribution(distributor, order_cycle)
               click_link 'Ship Order'
 
               within ".reveal-modal" do
+                expect(page).to have_checked_field(
+                  'Send a shipment/pick up notification email to the customer.'
+                )
                 uncheck 'Send a shipment/pick up notification email to the customer.'
                 expect {
                   find_button("Confirm").click
+
+                  # the best & easiest way to close the modal without calling all the JS
+                  find_button("Cancel").click
                 }.not_to enqueue_job(ActionMailer::MailDeliveryJob)
               end
 
+              click_link('Order Details')
               expect(order.reload.shipped?).to be true
               expect(page).to have_text 'SHIPPED'
             end
           end
         end
 
         it_behaves_like "ship order from dropdown", "Order Details"
-        context "pending examples" do
-          before { pending("#12369") }
-          it_behaves_like "ship order from dropdown", "Customer Details"
-          it_behaves_like "ship order from dropdown", "Payments"
-          it_behaves_like "ship order from dropdown", "Adjustments"
-          it_behaves_like "ship order from dropdown", "Invoices"
-          it_behaves_like "ship order from dropdown", "Return Authorizations"
+        it_behaves_like "ship order from dropdown", "Customer Details"
+        it_behaves_like "ship order from dropdown", "Payments"
+        it_behaves_like "ship order from dropdown", "Adjustments"
+        it_behaves_like "ship order from dropdown", "Invoices"
+        it_behaves_like "ship order from dropdown", "Return Authorizations"
+
+        context 'when not on the Order Details sub section' do
+          before do
+            click_link 'Customer Details'
+          end
+          it 'can ship order too' do
+            find('.ofn-drop-down').click
+            click_link 'Ship Order'
+
+            within ".reveal-modal" do
+              expect(page).to have_checked_field('Send a shipment/pick up ' \
+                                                 'notification email to the customer.')
+              # no test of enqueued job since it can cause failures
+              # if the remainder of the spec is too fasr
+              find_button('Confirm').click
+            end
+
+            # the best & easiest way to close the modal without calling all the JS
+            find_button("Cancel").click
+            expect(order.reload.shipped?).to be true
+            click_link('Order Details')
+            expect(page).to have_text 'SHIPPED'
+          end
```

Comment:
I think that this is now covered by the above shared examples. I just left it here for reference.

Reply:
Yes, it is now included in the shared examples, so I deleted this part. Also I modified a bit the shared examples, see below.